apps/web/src/app/[locale]/shop/BackgroundSection/BackgroundSection.tsx

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable @next/next/no-img-element */
+'use client';
+
+import { useTranslations } from 'next-intl';
+import { css, cx } from '_panda/css';
+import { renderUserQueries, shopQueries, useBuyBackground } from '@gitanimals/react-query';
+import { Button } from '@gitanimals/ui-panda';
+import { wrap } from '@suspensive/react';
+import { useQuery } from '@tanstack/react-query';
+import { AxiosError } from 'axios';
+import { toast } from 'sonner';
+
+import { useClientUser } from '@/utils/clientAuth';
+import { getBackgroundImage } from '@/utils/image';
+
+export const BackgroundSection = wrap
+  .ErrorBoundary({
+    fallback: (props) => (
+      <div>
+        <h2>An error occurred while loading backgrounds</h2>
+        <p>{props.error.message}</p>
+      </div>
+    ),
+  })
+  .Suspense({ fallback: <></> })
+  .on(function BackgroundSection() {
+    const t = useTranslations('Shop.Background');
+    const { name } = useClientUser();
+    const { data } = useQuery(shopQueries.backgroundOptions());
+    const { data: myBackground } = useQuery(renderUserQueries.getMyBackground(name));
```

Comment:
편안 ~

```suggestion
    const { data: background } = useQuery(shopQueries.backgroundOptions()); 
    const { data: myBackground } = useQuery(renderUserQueries.getMyBackground(name));
```

얘도 아래에서 읽기 쉽게 이름을 변경해줘도 좋을 거 같다는 생각인데, 갠적인 의견입니다!

Reply:
좋은 생각이네요!! 더 명확하게 보이면 좋을 것 같아요. 
background 보다는 `backgroundList` ..? 이렇게 변경해두겠습니다

Reply:
`shopBackgrond` 으로 하기로 결정하였어요. 
왜냐하면 `shopBackgrond.background`를 접근해야하더라구요

Reply:
ㅋㅋㅋㅋ좋아요!!

susepnse 쿼리를 이용한다면, `{ data: {background} }` 이렇게 접근해도 좋겠다는 생각이 들긴 하네요 🤔

Reply:
오... 천젠대?